python/lsst/pipe/tasks/repairPostDiff.py

```diff
@@ -0,0 +1,268 @@
+# This file is part of pipe_tasks.
```

Comment:
I had originally made a new one because I wanted to pull out all of the interpolation stuff. Looking at it more, though, indeed we could probably get away with using the existing repair task and just calling it with doInterp = False.